src/lib/ledger_catchup/ledger_catchup.ml

```diff
@@ -261,10 +261,11 @@ module Make (Inputs : Inputs.S) :
               Strict_pipe.Writer.write catchup_breadcrumbs_writer trees
               |> Deferred.return
         | Error e ->
-            Logger.info logger ~module_:__MODULE__ ~location:__LOC__
-              !"All peers either sent us bad data, didn't have the info, or \
-                our transition frontier moved too fast: %s"
-              (Error.to_string_hum e) ;
+            Logger.warn logger ~module_:__MODULE__ ~location:__LOC__
+              !"Catchup failed: All peers either sent us bad data, didn't \
```

Comment:
`Catchup process failed -- unable to receive valid data from peers or transition frontier progressed faster than catchup data received. See error for details: $error`